pkg/utils/engineHelper_test.go

```diff
@@ -0,0 +1,358 @@
+package utils
+
+import (
+	"reflect"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	litmuschaosv1alpha1 "github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	"github.com/litmuschaos/chaos-runner/pkg/log"
+)
+
+func TestSetExpImageFromEngine(t *testing.T) {
+	fakeNewExperimentImage := "fake-new-experiment-image"
+	var experiment ExperimentDetails
```

Comment:
do the similar initialization suggest in the previous file. repeat for all places